src/main/kotlin/org/rust/ide/annotator/RsCfgDisabledCodeAnnotator.kt

```diff
@@ -8,27 +8,23 @@ package org.rust.ide.annotator
 import com.intellij.ide.annotator.AnnotatorBase
 import com.intellij.lang.annotation.AnnotationHolder
 import com.intellij.lang.annotation.HighlightSeverity
-import com.intellij.openapi.util.TextRange
 import com.intellij.openapiext.isUnitTestMode
 import com.intellij.psi.PsiElement
 import org.rust.ide.colors.RsColor
 import org.rust.lang.core.psi.ext.RsDocAndAttributeOwner
 import org.rust.lang.core.psi.ext.isEnabledByCfgSelf
-import org.rust.lang.core.psi.ext.rangeWithSurroundingLineBreaks
 
 class RsCfgDisabledCodeAnnotator : AnnotatorBase() {
     override fun annotateInternal(element: PsiElement, holder: AnnotationHolder) {
         if (holder.isBatchMode) return
+
         if (element is RsDocAndAttributeOwner && !element.isEnabledByCfgSelf) {
-            holder.createCfgDisabledAnnotation(element.rangeWithSurroundingLineBreaks)
-        }
-    }
+            val color = RsColor.CFG_DISABLED_CODE
+            val severity = if (isUnitTestMode) color.testSeverity else HighlightSeverity.INFORMATION
 
-    private fun AnnotationHolder.createCfgDisabledAnnotation(textRange: TextRange) {
-        val color = RsColor.CFG_DISABLED_CODE
-        val severity = if (isUnitTestMode) color.testSeverity else HighlightSeverity.INFORMATION
-        // BACKCOMPAT: 2019.3
-        @Suppress("DEPRECATION")
-        createAnnotation(severity, textRange, "Conditionally disabled code").textAttributes = color.textAttributesKey
+            holder.newAnnotation(severity, "Conditionally disabled code")
+                .textAttributes(color.textAttributesKey)
+                .create()
+        }
```

Comment:
These changes are not just using a new API, they lead to different behaviours
Old:
![image](https://user-images.githubusercontent.com/2539310/97453905-3ce5d600-1947-11eb-899d-5fc2839bc260.png)

New:
![image](https://user-images.githubusercontent.com/2539310/97453932-42432080-1947-11eb-81ef-c0636c05073a.png)


Is it intentional?

Reply:
`rangeWithSurroundingLineBreaks` was used before to handle background highlighting, but then we decided to use the foreground instead (there is a note in the commit body). So this is intentional. But actually, I hadn't thought about the color of comment highlighting. Do you think it is important?

Reply:
> Do you think it is important?

No, I don't think so